Added deepin version convention
--- a/make_version_h.pl
+++ b/make_version_h.pl
@@ -13,7 +13,9 @@
 my ($ver) = $line =~ /^whois \s+ \( ( [^\)]+ ) \) \s+ \S+/x;
 die "Version number not found in $changelog!\n" if not $ver;
 
-$ver =~ s/ ( ~bpo\d+\+\d+ | \+b\d+ | ~deb\d+.* | ubuntu\d+ | build\d+ | \+dyson\d+ ) $//x;
+$ver =~ s/ ( -deepin\d* ) //x;
+$ver =~ s/ ( \+u\d{3} ) //x;
+$ver =~ s/ ( ~bpo\d+\+\d+ | \+b\d+ | \+rb\d+ | ~deb\d+.* | ubuntu\d+ | build\d+ | \+dyson\d+ ) $//x;
 
 # The version number must not deviate from this format or the -V option
 # to RIPE-like servers will break. If needed, update the previous regexp.
